.github/workflows/conformance.yml

```diff
@@ -29,16 +29,21 @@ jobs:
           - kind
           - eks
         config:
-          - name: "consul@v1.11 + consul-k8s@v0.45.0"
+          - name: "consul@v1.11 + consul-k8s@v0.47.0"
             api-gateway-image: "hashicorppreview/consul-api-gateway:0.4-dev"
             consul-image: "hashicorp/consul:1.11"
             envoy-image: "envoyproxy/envoy:v1.20-latest"
-            consul-k8s-version: "charts/capigw-controller-clusterrole-referencegrants"
-          - name: "consul@v1.12 + consul-k8s@v0.45.0"
+            consul-k8s-version: "main"
+          - name: "consul@v1.12 + consul-k8s@v0.47.0"
             api-gateway-image: "hashicorppreview/consul-api-gateway:0.4-dev"
             consul-image: "hashicorp/consul:1.12"
             envoy-image: "envoyproxy/envoy:v1.22-latest"
-            consul-k8s-version: "charts/capigw-controller-clusterrole-referencegrants"
+            consul-k8s-version: "main"
+          - name: "consul@v1.13 + consul-k8s@v0.47.0"
+            api-gateway-image: "hashicorppreview/consul-api-gateway:0.4-dev"
+            consul-image: "hashicorp/consul:1.13"
+            envoy-image: "envoyproxy/envoy:v1.23-latest"
```

Comment:
Looks like we will likely be downgrading consul-k8s to envoy 1.22.4 before releasing v0.47.0

Reply:
```suggestion
            envoy-image: "envoyproxy/envoy:v1.22-latest"
```